libs/ui/src/error_boundary.tsx

```diff
@@ -28,6 +28,8 @@ export class ErrorBoundary extends React.Component<Props, State> {
   constructor(props: Props) {
     super(props);
     this.state = {};
+
+    this.handleUnhandledRejection = this.handleUnhandledRejection.bind(this);
```

Comment:
This is an important one for us to up-port to v4. I noticed errors in kiosk-browser because of this function's lack of binding when it's called. Causing the error boundary itself to hang in rare circumstances. Will document more fully in a standalone PR for v4!

Reply:
good find!